Resources/Textures/Structures/Machines/VendingMachines/smartfridge.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/1516a728931b4985c1e86f0c5995a5aa1554a1ad and modified by Swept",
+  "copyright": "Taken from /tg/station and modified by Swept & MeltedPixel",
```

Comment:
don't remove the copyright info

Reply:
Fair enough, but this copywrite doesn't point to anything that makes sense, Automatic changelog generation for PR is where it points.

Reply:
all that matters is that it points to a commit where the sprite was taken from